erp/models.py

```diff
@@ -395,6 +397,13 @@ class Meta:
         verbose_name="Courriel",
         help_text="Adresse email permettant de contacter l'ERP",
     )
+    private_contact_email = models.EmailField(
+        max_length=255,
+        null=True,
+        blank=True,
+        verbose_name="Courriel privé",
+        help_text="Email permettant de notifier le fournisseur en cas de modifications par un tier",
+    )
```

Comment:
As discussed this morning, we don't need this new field as we can already use the email address of the user account associated to the ERP entry for notifications.